windows/server_stats.ps1

```diff
@@ -80,6 +80,11 @@ $ServerStats = {
         $perf = @(getPerf)
         Start-Sleep -Seconds $CpuUtilizationTimeout
         $perf += getPerf
+
+        # Convert start and end timestamps from LDAP time
+        $CPUUtilization_StartTimeStamp = w32tm.exe /ntte $perf[0].TimeStamp_Sys100NS
```

Comment:
Didn't know we can directly use EXEs like this. Although power**SHELL** now makes sense. 😅

Reply:
Yeah it's cool, the alternative is to use the dotnet function `FromFileTimeUtc` with something like `[DateTime]::FromFileTimeUtc(TimeStamp-value)` but this doesn't work in this context unfortunately.